paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -781,4 +782,20 @@ class PaymentSheet internal constructor(
             }
         }
     }
+
+    companion object {
+        /**
+         * Deletes all persisted state.
```

Comment:
Can you make this function more clear as to what data is being cleared?  Seems like it is the data specfic to a user, but if that is the case it seems like the DefaultCardAccountRangeStore would also be cleared.

A comment describing what is and is not cleared will be helpful in the future so that we understand why some, but not all preferences are cleared.

Reply:
Documentation was discussed in the API review linked in this PR, feel free to bring up and suggest changes, but FYI the iOS version is already live: https://github.com/stripe/stripe-ios/commit/eb524bc244b24570f54370ada54ce36dbbcd3acd.
`DefaultCardAccountRangeStore` is not specific to a user, it behaves like a cache.